www/kohv04/hw08/main.js

```diff
@@ -0,0 +1,453 @@
+ * buttonSayHello.innerText = 'Say Hello';
+ * // nastavení atributu id tlačítka
+ * buttonSayHello.setAttribute('id', 'task-0');
+ * // nabindování funkce na událost click tlačítka
+ * buttonSayHello.addEventListener('click', sayHello);
+ * 
+ * // výběr existujícího elementu na stránce s id="tasks"
+ * const tasks = document.querySelector('#tasks');
+ * // vložení vytvořeného tlačítka do vybraného elementu na stránce
+ * tasks.appendChild(buttonSayHello);
+ */
+// Solution here
+const task3 = () => {
+    console.log("Funkce function fonction funktio. Vemte předchozí úlohy a udělejte z nich funkce. Tj. vytvoříte funkce, které přijímají argumenty, a na základě argumentů po zavolání vypíše výsledek na konzoli. Párkrát zavolejte tyto funkce s různými argumenty. V konzoli také vyzkoušejte, zda fungují vaše funkce. ");
+};
+const Button3 = document.createElement('button');
+Button3.innerText = 'Uloha 3 (Funkce function fonction funktio)';
+Button3.setAttribute('id', 'task-3');
+Button3.addEventListener('click', task3);
+tasks.appendChild(Button3);
+
+
+
+
+
+console.log("They were already made as functions.");
+const Giuseppe = {
+    name: "Giuseppe",
+    birth: "1998",
+};
+const Gianna = {
+    name: "Gianna",
+    birth: "2002",
+};
+sentence1(Giuseppe);
+sentence1(Gianna);
+
+const CELSIUS2 = 0;
+const FAHRENHEIT2 = 451;
+FROM_CELSIUS_TO_FAHRENHEIT(CELSIUS2);
+FROM_FAHRENHEIT_TO_CELSIUS(FAHRENHEIT2);
+const CELSIUS3 = 100;
+const FAHRENHEIT3 = -459;
+FROM_CELSIUS_TO_FAHRENHEIT(CELSIUS3);
+FROM_FAHRENHEIT_TO_CELSIUS(FAHRENHEIT3);
+
+const Button_R3 = document.createElement('button');
+Button_R3.innerText = 'Reseni Uloha 3 (Funkce function fonction funktio)';
+Button_R3.setAttribute('id', 'result-3');
+Button_R3.addEventListener('click', sentence1);
+Button_R3.addEventListener('click', FROM_CELSIUS_TO_FAHRENHEIT);
+Button_R3.addEventListener('click', FROM_FAHRENHEIT_TO_CELSIUS);
+tasks.appendChild(Button_R3);
+
+/**
+ * 4) %CENSORED%. Vytvořte funkci, která vezme 2 číselné argumenty a vrátí podíl prvního čísla a druhého čísla. 
+ * Výsledek vypište v procentech do předem vytvořeného místa na stránce pro výsledky, např. 21 je 50% z 42. Pro 
+ * zkrácení / zaokrouhlení desetinných míst použijte funkci .toFixed(n). Např. var pi = 3.1415926535; pi.toFixed(2); 
+ * Pozor na dělení nulou! 
+ * 
+ * Pro testování vytvořte tlačítko s touto funkcí podle pokynu v úloze č. 3
+ */
+// Solution here
+const task4 = () => {
+    console.log("Vytvořte funkci, která vezme 2 číselné argumenty a vrátí podíl prvního čísla a druhého čísla. Výsledek vypište v procentech do předem vytvořeného místa na stránce pro výsledky, např. 21 je 50% z 42. Pro zkrácení / zaokrouhlení desetinných míst použijte funkci .toFixed(n). Např. var pi = 3.1415926535; pi.toFixed(2); Pozor na dělení nulou!");
+};
+const Button4 = document.createElement('button');
+Button4.innerText = 'Uloha 4 (%CENSORED%)';
+Button4.setAttribute('id', 'task-4');
+Button4.addEventListener('click', task4);

```

Comment:
vzdyt kdyz na ty tlacitka kliknu tak se mi vypise jenom ten text zadani ulohy.

Reply:
Ano, strávil jsem nad tím pár hodin, a pak se smířil s tím, že jednoduše nevím jak na to. Možná na to v budoucnu přijdu. Doufám, že to bude stačit aspoň na 0,5 bodu.